substrate/frame/revive/src/benchmarking/mod.rs

```diff
@@ -886,6 +886,23 @@ mod benchmarks {
 		assert_eq!(U256::from_little_endian(&memory[..]), runtime.ext().block_number());
 	}
 
+	#[benchmark(pov_mode = Measured)]
+	fn seal_block_author() {
+		build_runtime!(runtime, memory: [[123u8; 20], ]);
+		let result;
+		#[block]
+		{
+			result = runtime.bench_block_author(memory.as_mut_slice(), 0);
+		}
+		assert_ok!(result);
+		let block_author = runtime
+			.ext()
+			.block_author()
+			.map(|account| T::AddressMapper::to_address(&account))
+			.unwrap_or(H160::zero());
```

Comment:
Can we somehow make sure that during the benchmark we are not running into this case? I assume this happens when the `Digest` storage item is empty.

Reply:
Yes need to shed some light on this. I'll peek at the pallet_authorship or somewhere else to see how to properly benchmark this.

Reply:
I suggest asking Basti to find out if it is a lot of work. I would really like to test the `Some` case here to make sure it is actually working.

Reply:
Updated the benchmark to:
- Overestimate the opcode cost to get safe numbers
- Return a value in `BABE` and `AURA` consensus configuration